test/browser/pages/confirm.js

```diff
@@ -12,7 +12,11 @@ module.exports = class PlaywrightDevPage {
   }
 
   async confirmDetails() {
-    await this.page.click(".govuk-button");
+    await this.page.locator(":nth-match(.govuk-button, 2)");
```

Comment:
Once we have a confirmed UI, this can be targeted with `data-id` attributes instead.

Reply:
@merlinc I'll add a code TODO for this so it doesnt get left behind.